src/canisters/frontend/ic-certified-assets/src/state_machine.rs

```diff
@@ -574,17 +613,28 @@ impl From<StableState> for State {
             ..Self::default()
         };
 
-        for (asset_name, asset) in state.assets.iter_mut() {
-            for enc in asset.encodings.values_mut() {
-                enc.certified = false;
+        let assets_keys: Vec<_> = state.assets.keys().map(|key| key.clone()).collect();
```

Comment:
This clone is not nice, but I can't find a better way to satisfy the borrow checker